### DISCUSSION TITLE
Make "multiple" as optional attribute on <x-media-library-collection>

### DISCUSSION THREAD

**Discussion by levipeto** [spatie/laravel-medialibrary#2908] [Ideas]
Hello,
I understand livewire multiple file upload limitations to S3. However, the component could be still helpful if the "multiple" attribute was not hardcoded but an attribute with a default "true" value:

Current:
<livewire:media-library
    :key="'media-library-component' . $name"
    :media="$media"
    :model="$model"
    :name="$name"
    :rules="$rules"
    :maxItems="$maxItems"
    **:multiple="true"**
    :sortable="$sortable"
    :view="$componentView ?? null"
    :listView="$listView"
    :itemView="$itemView"
    :propertiesView="$propertiesView ?? null"
    :fieldsView="$fieldsView ?? null"
    :editableName="$editableName"
/>

If it was something like  ":multiple=$multiple ?? true."

Then we could still upload multiple files, but not simultaneously, which is ok for now, but currently, this makes that component unusable for S3.

"Multiple" could be still turned off but the upload and the collection would still work.
<x-media-library-collection
                            **:multiple="false"**
                            :name="$name"
                            :model="$product"
                            :collection="$name"
                    />

**Comment by freekmurze** [comment]
Hi @levipeto 

Feel free to PR that!

**Reply by levipeto** [reply]
I just realized that's a feature for the **pro** version. Ready to push the changes and create PR, but I don't have write access to the repo. 
It's 3 lines of code.
